google/cloud/speech/v1beta1/cloud_speech_gapic.yaml

```diff
@@ -19,6 +19,13 @@ license_header:
   license_file: license-header-apache-2.0.txt
 interfaces:
 - name: google.cloud.speech.v1beta1.Speech
+  smoke_test:
+    method: SyncRecognize
+    init_fields:
+      - config.language_code="en-US"
```

Comment:
the lines starting with '-' should have the same number of spaces as the key line (so two spaces in from where you have it)